numba_dpex/dpctl_iface/usm_allocators_ext.c

```diff
@@ -50,8 +50,6 @@ void *save_queue_allocator(size_t size, void *opaque)
     // Use that queue to allocate.
     void *data = (void *)DPCTLmalloc_shared(new_size, cur_queue);
     if (data == NULL) {
-        fprintf(stderr,
```

Comment:
does it complain if we have std::cerr?

Reply:
Tried something like this: 
```

fprintf(stderr, "Fatal: failed to allocate memory for AllocatorImpl.\n");
```
Checkmarx doesn't like it: Improper Resource Access Authorization.

Reply:
std::cerr need to include <iostream> from cpp, maybe not mix c++ and c just for this output?